lib/pronto/credo_runner.rb

```diff
@@ -10,7 +10,7 @@ def initialize(_, _)
     def run
       return [] unless @patches
 
-      compile
+      compile if ENV["PRONTO_CREDO_COMPILE"]
```

Comment:
great! could you add a value to verify? for example: `compile if(ENV["PRONTO_CREDO_COMPILE"] == 1)`